cmd/stability/differ.go

```diff
@@ -0,0 +1,77 @@
+package stability
+
+import (
+	"fmt"
+	"log"
+
+	pb "github.com/apigee/registry/rpc"
+	"github.com/getkin/kin-openapi/openapi3"
+	"github.com/golang/protobuf/proto"
+	"github.com/tufin/oasdiff/diff"
+	"gopkg.in/yaml.v3"
+)
+// tuffin/oasdiff options
+type DiffOptions struct{
+	prefix string
+	filter string
+	diffFormat string
+	excludeExamples bool
+	excludeDescription bool
+	summary bool
+}
+
+func NewDefaultDiffOptions() DiffOptions{
+	diffOptions := DiffOptions{}
+	diffOptions.excludeExamples = false
+	diffOptions.excludeDescription = false
+	diffOptions.summary = true
+	return diffOptions
+}
+//Diff 2 specs with set diff options.
+func (diffOptions *DiffOptions)GetDiff(base []byte, revision []byte) ([]byte, error){
+	loader := openapi3.NewLoader()
+	loader.IsExternalRefsAllowed = true
+	spec1, err := loader.LoadFromData(base)
+	if err != nil {
+		fmt.Printf("failed to load base spec from %q with %v", base, err)
+		return nil, err
+	}
+	spec2, err := loader.LoadFromData(revision)
+	if err != nil {
+		fmt.Printf("failed to load revision spec from %q with %v", revision, err)
+		return nil, err
+	}
+	diffReport, err := diff.Get(&diff.Config{
+		ExcludeExamples:    diffOptions.excludeExamples,
+		ExcludeDescription: diffOptions.excludeDescription,
+		PathFilter:         diffOptions.filter,
+		PathPrefix:         diffOptions.prefix,
+	}, spec1, spec2)
+	if err != nil {
+		fmt.Printf("diff failed with %v", err)
+		return nil, err
+	}
+	var yaml_bytes = getYAML(diffReport)
+	//Convert Yaml Diff to string.
+	stringYamlDiff := string(yaml_bytes)
```

Comment:
This comment seems to just restate what the code is doing. Code comments are great for explaining _why_ something is happening, while the code itself should be understandable enough to explain _what_ is happening without commentary.

Since the line of code is so clear already and the variable is only used in one location, consider doing the conversion in-line when you set the struct field instead.